util/libcrypto.num

```diff
@@ -4501,3 +4501,13 @@ EVP_sha512_256                          4442	1_1_1	EXIST::FUNCTION:
 EVP_sha512_224                          4443	1_1_1	EXIST::FUNCTION:
 OCSP_basic_sign_ctx                     4444	1_1_1	EXIST::FUNCTION:OCSP
 RAND_DRBG_bytes                         4445	1_1_1	EXIST::FUNCTION:
+SM2_do_sign                             4446	1_1_1	EXIST::FUNCTION:
+SM2_do_verify                           4447	1_1_1	EXIST::FUNCTION:
+SM2_ciphertext_size                     4448	1_1_1	EXIST::FUNCTION:
+SM2_plaintext_size                      4449	1_1_1	EXIST::FUNCTION:
+SM2_encrypt                             4450	1_1_1	EXIST::FUNCTION:
+SM2_decrypt                             4451	1_1_1	EXIST::FUNCTION:
+SM2_compute_userid_digest               4452	1_1_1	EXIST::FUNCTION:
+SM2_verify                              4453	1_1_1	EXIST::FUNCTION:
+SM2_sign                                4454	1_1_1	EXIST::FUNCTION:
+ERR_load_SM2_strings                    4455	1_1_1	EXIST::FUNCTION:
```

Comment:
Do these actually have to be exported? And if actually so, does it mean that no-sm2 wouldn't work? And it seems that it doesn't. I mean attempt to configure with no-sm2 fails. Is it actually intentional?

Reply:
See #5670.